transform.py

```diff
@@ -1 +1,121 @@
+from datetime import datetime
+
+from pyspark.sql import SparkSession, Row
+from pyspark.sql.functions import lit
+from pyspark.sql.types import StructType, StructField, StringType, FloatType, ArrayType, DateType
+
 from kafka_interface import kafka_interface
+
+
+class Transform:
+    def __init__(self, kafka_topics, spark_master="local[*]"):
```

Comment:
spark or flink? what do you prefer to use?
If you want to use python naitively it's ok as well